reference/glossary.md

```diff
@@ -4,13 +4,17 @@ This glossary defines the terminology used to describe the various concepts rela
 
 **Actuator**: An *actuator* is a *node* representing a real robotics actuator such as a motor, a LED, a radio emitter, etc. *Actuators* may receive commands from a *controller*. Their behavior will affect the outcome of a simulation. Some *actuators* are also *sensors* if they make sensory measurement, such as a motor than can provide position or force feedback.
 
+**Ancestor node**: An *ancestor node* is a *node* that hierarchically includes another *node* in the *scene tree*.
+

```

Comment:
I would rephrase it relatively to the descendant node, e.g.:

**Ancestor**: The *ancestor* of a *descendant* is a *node* which hierarchically contains the *descendant* in the *scene tree*.